.github/workflows/lint.yml

```diff
@@ -6,6 +6,18 @@ jobs:
   pre-commit:
     runs-on: ubuntu-latest
     steps:
+      - name: Free Up GitHub Actions Ubuntu Runner Disk Space 🔧
+        uses: jlumbroso/free-disk-space@main
+        with:
+          # This might remove tools that are actually needed, if set to "true" but frees about 6 GB
+          tool-cache: false
+
+          # All of these default to true, but feel free to set to "false" if necessary for your workflow
+          android: true
+          dotnet: true
+          haskell: true
+          large-packages: true
+          swap-storage: true
```

Comment:
whats going on here?

Reply:
I was having trouble getting actions to run due to the size of the repository, so I needed to free up space, that is why I added that action, I would need to remove it and test again to see if the space problem continues

Reply:
Is this a recent issue? We dealt with this a long time ago, but I haven't seen it recently?

Reply:
It's been a while since we last updated opinionated, I'm not sure if the problem persists